.github/workflows/main.yml

```diff
@@ -27,17 +27,17 @@ jobs:
         env: 
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - run: ./scripts/build_image.sh
-        if: matrix.os == 'ubuntu-latest' && (github.ref == 'refs/heads/main')
+        if: matrix.os == 'ubuntu-latest' && (github.ref == 'refs/heads/master' || github.ref == 'refs/heads/develop')
```

Comment:
There is something funky happening here. The new version is the wrong version :)